server/controllers/bookController.js

```diff
@@ -227,6 +227,54 @@ class BookController {
     await models.BorrowedBooks.update({ returned: true }, { where: { isbn } });
     return Utils.successStatus(res, 200, 'Book successfully returned');
   }
+
+  static async reserveBook(req, res) {
+
+    const { loggedinUser } = req;
+
+    const { isbn } = req.query;
+
+    const { title } = await models.Books.findOne({ where: { isbn }});
+
+    let expiryDate = new Date();
+
+    expiryDate.setDate(expiryDate.getDate() + 3);
+
+    const reservedBook = await models.reservedBooks.create(
+      {
+        title, 
+        isbn, 
+        patronId: loggedinUser.id, 
+        collected: false,
+        timeToExpire: expiryDate
+      });
+
+    return Utils.successStatus(res, 200, 'Book reserved successfully', {
+      id: reservedBook.id,
+      title,
+      isbn,
+      patronId: reservedBook.patronId,
+      collected: reservedBook.collected,
+      Expires: reservedBook.timeToExpire.toDateString(),
```

Comment:
What do you use the value for?
And why is it in Title case?

Reply:
Title case was a mistake. If I get your question, you want to know what I use the value of the expires property for? It's so a reserved book can expire for another user to be able to either reserve or borrow it

Reply:
But you do not use it when search for a reserved book. So I expected you to find reserved book where ID and not expired... something like that